Makefile

```diff
@@ -136,6 +139,22 @@ INSTALL     ?= $(EXES)
 INSTALLEES  := \
   $(filter $(INSTALL),$(filter-out $(DONT_INSTALL),$(EXES) uutils))
 
+# Shared library extension
+SYSTEM := $(shell uname)
+DYLIB_EXT := 
+ifeq ($(SYSTEM),Linux)
+	DYLIB_EXT    := so
```

Comment:
I'd prefer that `DYLIB_EXT` include the `.`, so it would be like `DYLIB_EXT := .so`.